vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c

```diff
@@ -600,7 +600,24 @@ BTStatus_t prvBTStartService( uint8_t ucServerIf,
                               uint16_t usServiceHandle,
                               BTTransport_t xTransport )
 {
-    return eBTStatusUnsupported;
+    BTStatus_t xStatus = eBTStatusFail;
+
+    if ( serviceCnt != 0 )
+    {
+        /* GATT Service is supposed to start in prvAddServiceBlob, we just
+         * check if handle is matching with the one populated in prvAddServiceBlob */
+        if ( usServiceHandle == afrServices[ serviceCnt - 1 ]->pusHandlesBuffer[ 0 ] )
```

Comment:
To be safe, will it be better to loop through all started service handles and check if `usServiceHandle` matches any of them?

Reply:
Thanks, makes sense !!

Reply:
Done !!